README.md

```diff
@@ -67,16 +66,16 @@ script that starts the connector. Here is an example for python:
 ```
 
 To run your connector as part of a Conduit pipeline you can create it using the
-connectors API and specify the path to the compiled connector binary in the
-field `plugin`.
+connectors API and use the `standalone:connector-name` in the field `plugin`, where
```

Comment:
Maybe also mention that the plugin executable needs to be placed in the `connectors` folder so Conduit loads it on startup.